release/models/devices-manifest/openconfig-terminal-device-properties.yang

```diff
@@ -0,0 +1,501 @@
+      filter characterization, CD and DGD tolerance.";
+
+  oc-ext:openconfig-version "0.1.0";
+
+
+  // Revisions
+  revision "2021-06-21" {
+      description "Initial manifest fine to extend the information
+      related to the operational modes supported by a terminal device";
+      reference "0.1.0";
+  }
+
+
+  // OpenConfig specific extensions for module metadata.
+  oc-ext:regexp-posix;
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig-properties";
+
+  // grouping statements
+
+  grouping standard-mode-features-description {
+    description
+      "Standard mode features description grouping. It is used if the
+      'mode-type' attribute is set to 'TRANSCEIVER_MODE_TYPE_STANDARD";
+
+    container G.698.2 {
+      leaf standard-mode {
+        type oc-opt-term-prop-types:standard-mode;
+        description
+          "G.698.2 (11/18) standard mode";
+      }
+      description
+        "ITU-T G.698.2 (11/18) standard mode that guarantees interoperability.
+        It must be an string with the following format:
+        B-DScW-ytz(v) where all these attributes are conformant
+        to the ITU-T G.698.2 (11/18) recomendation.";
+    }
+  }
+
+  grouping explicit-mode-features-description {
+    description
+      "Definition of propiertary or non-standard operational-modes, which can be
+      uniformly characterized by the set of attributes included in their
+      operational-mode-capabilities which defines the related signal physical
+      impairment related aspects such Rx and Tx associated attibutes and tolerances,
+      ; and its optical-channel-config-value-constrains, which defines what are the
+      allowed values to be configured at the oc-component:optical-channel instance
+      configured to this mode.";
+
+    container explicit-mode {
+      description
+        "Explicit defition of the operational-mode. Typically this is used
+        for non-standard/propietary modes defined by the terminal-device
+        vendor and it is self described by the capabilities included in
+        the subtree underneath.";
+
+      container operational-mode-capabilities {
+        description
+          "Set of attributes which charaterizes the operational-mode for optimal
+          optical-channel transmission and receiver functions. This attributes
+          are intending to describe all the relevant data used during the
+          network planning phase, to estimate the physical-impairment
+          tolerances which can be introduced by the DWDM optical path,
+          while assuring optimal transmission function.";
+
+        uses operational-mode-capabilities-top;
+      }
+      container optical-channel-config-value-constrains{
+        description
+          "Set of constrains of the configuration attributes
+          of the optical-channel associated to the selected
+          operational-mode.";
+
+        uses config-value-constrains-top;
```

Comment:
s/constrains/constraints

Actually please check throughout the file for this.

Reply:
Full grammar and typos check done, latest commits includes all fixes.